tssh/tools.go

```diff
@@ -459,6 +461,25 @@ func execLocalTools(argv []string, args *sshArgs) (int, bool) {
 		return execEncodeSecret()
 	case args.NewHost || len(argv) == 0 && isFileNotExistOrEmpty(userConfig.configPath):
 		return execNewHost(args)
+	case args.ListHosts:
```

Comment:
Please make a new function `execListHosts` in `tools_list_hosts.go`. You can refer to `tools_enc_secret.go`.